lib/charts.dart

```diff
@@ -48,18 +48,13 @@ abstract class Chart {
     jsChart.callMethod('clearChart');
   }
 
-  static Future load({List<String> packages, String version: "current"}) {
+  static Future load({List<String> packages, String version: "current", String language}) {
```

Comment:
language should have a default value or be before the version parameter!

Reply:
No default value must be set since the js lib will try to find the best match for the language option when no language option is provided. Wasn't aware optional parameters which do not have a default values should be placed before those which do. Will change.